src/hisp/plamsa_data_handling/main.py

```diff
@@ -20,69 +22,82 @@ def __init__(
         self._time_to_RISP_data = {}
 
     def get_particle_flux(
-        self, pulse_type: str, nb_mb: int, t_rel: float, ion=True
+        self, pulse_type: str, nb_bin: SubBin | DivBin, t_rel: float, ion=True
     ) -> float:
         """Returns the particle flux for a given pulse type
 
         Args:
             pulse_type: pulse type (eg. FP, ICWC, RISP, GDC, BAKE)
-            nb_mb: monoblock number
+            bin: SubBin or DivBin
```

Comment:
you have `nb_bin` in the arguments list, but `bin` in the docs.

Since the type of this argument is a `SubBin` or a `DivBin` then I think it should be `bin` and not `nb_bin`.